src/views/DissolutionFirm/DissolutionFirm.vue

```diff
@@ -328,6 +330,10 @@ export default class DissolutionFirm extends Mixins(DateMixin) {
         ${this.dateToPacificDate(this.startDateMax, true)}`
     ]
   }
+
+  private startDateHandler (event: string): void{
+    this.dissolutionDate = event
+  }
```

Comment:
This is correct code, but if there will be no other code here, you can just do the assignment in the template, eg `@emitDateSync="dissolutionDate = $event"`.

I leave this up to you.